### PR TITLE
Add basic models, infra and tests for users service

### PR DESCRIPTION
**Pull request by dedque8**
Testing: go test ./... -v

### CODE REVIEW

File: users_service/.gitignore
Comment (Slowpokew) [inline users_service/.gitignore:1]: Should we make common gitignore for all services?

File: users_service/go.mod
Comment (Slowpokew) [inline users_service/go.mod:1]: поясниш за це

File: users_service/.gitignore
Comment (ladaguedencre) [reply users_service/.gitignore:1]: I created it in second PR, but included only very basic files not dependant on language e.t.c.

Comment (ladaguedencre) [reply users_service/.gitignore:1]: For example `*.out` may not be applicable for other services

File: users_service/models/user.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package models
+
+type User struct {
+	Id       string
```
Comment (b1rr0) [inline users_service/models/user.go]: https://github.com/google/uuid

File: users_service/.gitignore
Comment (dedque8) [reply users_service/.gitignore:1]: Done

**Review by Slowpokew — APPROVED** [review]

**Review by b1rr0 — CHANGES_REQUESTED** [review]

File: users_service/models/user.go (hunk 1 above)
Comment (dedque8) [reply users_service/models/user.go]: Used

**Review by b1rr0 — APPROVED** [review]

**Pull request merged**